### ISSUE TITLE
Process does not fail if Chan gelod.md is not found

### ISSUE THREAD

**Issue by rgreinho** [rgreinho/keeparelease#24]

**Comment by rgreinho** [comment]
For instance:

```bash
Running target: Release
Running dependency: Publish
exec: keeparelease -f CHANGELOG.md -a /home/circleci/project/dist/keeparelease-1.2.2-darwin-amd64 -a /home/circleci/project/dist/keeparelease-1.2.2-linux-amd64 -a /home/circleci/project/dist/keeparelease-1.2.2-windows-amd64
Error creating release: Unprocessable Entity (HTTP 422)
tag_name is not a valid tag
tag_name is not well-formed
Published releases must have a valid tag
CircleCI received exit code 0
```

Same thing if the Changelog is not found.

**Comment by rgreinho** [comment]
Fixed in #27.
